gradle/dependencies.gradle

```diff
@@ -141,7 +141,6 @@ versions += [
   // has the version field as mandatory in its configuration, see
   // https://github.com/scalameta/scalafmt/releases/tag/v3.1.0.
   scalafmt: "3.7.14",
-  scalaJava8Compat : "1.0.2",
```

Comment:
Could you please also update `LICENSE-binary`?

https://github.com/apache/kafka/blob/76a1af984b39d9890fe26954aff36bb1a321af77/LICENSE-binary#L262

Reply:
Done.
Thanks for pointing out. 😸